app/api/v1/helpers/crud_helpers.rb

```diff
@@ -0,0 +1,13 @@
+module V1
+  module Helpers
+    module CrudHelpers
+      def sorting(class_name, sort_by, default_sort)
```

Comment:
I'd control `sort_by`, cause it's quite unpredictable what can be passed to a query

I believe that's possible to get all column names of entities for checking if it includes `sort_by` value